app/admin/simulator/page.tsx

```diff
@@ -0,0 +1,133 @@
+export default function SimulatorPage() {
+  return (
+    <div className="flex flex-col gap-6">
+      <div>
+        <h1 className="text-2xl font-bold mb-2">App Simulator</h1>
+        <p className="text-sm text-gray-500">Simulate preview environments for different applications</p>
+      </div>
+
+      <div className="bg-white border border-gray-300 rounded-lg">
+        <div className="bg-gray-500 px-6 py-3 rounded-t-lg">
+          <h2 className="text-lg font-semibold text-white">Slack Simulator</h2>
```

Comment:
Organize this page almost like a smartphone Home Screen, with multiple app icons. Clicking on one would take the user to /admin/simulator/slack, for example